pkg/cloudprovider/kubevirt/types/types.go

```diff
@@ -18,70 +18,13 @@ package types
 
 import (
 	providerconfigtypes "github.com/kubermatic/machine-controller/pkg/providerconfig/types"
-
-	corev1 "k8s.io/api/core/v1"
 )
 
 type RawConfig struct {
-	Auth           Auth           `json:"auth,omitempty"`
-	VirtualMachine VirtualMachine `json:"virtualMachine,omitempty"`
-	Affinity       Affinity       `json:"affinity,omitempty"`
+	Auth Auth `json:"auth,omitempty"`
 }
 
 // Auth
 type Auth struct {
 	Kubeconfig providerconfigtypes.ConfigVarString `json:"kubeconfig,omitempty"`
 }
-
```

Comment:
What happened here? Those are not raw config anymore?

Reply:
We don't need those parts. We only consume fields that are adequate enough to generate cloud config.